release-notes/3.0/3.0.0-preview2.md

```diff
@@ -0,0 +1,102 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 3.0.0-preview2.21105.01 released 15 April 2021
+
+This update brings the below changes over the previous release:
+
+### Breaking Changes over Preview V3.0.0-preview1
+- `User Id` now accepts `Client Id` instead of `Object Id` for **User-Assigned Managed Identity**.
+- **SqlDataReader** now returns a `DBNull` value instead of an empty `byte[]`. To active the legacy behaviour, you must enable the following AppContext switch: 
```

Comment:
```suggestion
- **SqlDataReader** now returns a `DBNull` value instead of an empty `byte[]`. To active the legacy behavior, you must enable the following AppContext switch: 
```